app/assets/stylesheets/application.scss

```diff
@@ -30,6 +30,27 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+
+ .green-completion{
+  color: white;
+  width: 85px;
+  background-color: green;
+  margin: 5px 5px 5px 5px;
+  border-width: 1px;
+  border-style: solid;
+  border-color: rgba(0,0,0,.2);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.2)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`.2` should be written with a leading zero as `0.2`<br>Commas in function arguments should be followed by a single space